packages/accounts-controller/src/AccountsController.ts

```diff
@@ -965,6 +986,11 @@ export class AccountsController extends BaseController<
     accountId: string,
   ): AccountsControllerState['internalAccounts']['accounts'] {
     delete accountsState[accountId];
+
+    this.messagingSystem.publish(
+      'AccountsController:accountRemoved',
+      accountId,
+    );
```

Comment:
Nit: (to follow the same pattern than for `accountAdded`) 
```suggestion
    );
    
```